aten/src/ATen/native/native_functions.yaml

```diff
@@ -3363,12 +3363,12 @@
 - func: _sparse_softmax(Tensor self, int dim, bool half_to_float) -> Tensor
   use_c10_dispatcher: full
   dispatch:
-    SparseCPU: softmax_sparse_cpu
+    SparseCPU, SparseCUDA: softmax_sparse
```

Comment:
I think you're going to watch to separate these dispatches (here and for the other functions)

Reply:
all right!